### ISSUE TITLE
Button focus is not clearly visible (blue on blue)

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52676]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 2.4.7

**Repro Steps:**
1. Navigate to Status Bar at the bottom and move to notifications.
2. Move to any notification that appears at the screen.
3. Navigate to any button inside the notification.

**Actual:**
The focus is not marked using dotted-lines/rectangle to indicate the focus on the buttons. Only color changes to mark the focus on the buttons.

**Expected:**
The focus should be visible using dotted-lines/rectangle to indicate the focus on the buttons. 

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
Low vision/visually impaired users should be able to distinguish the focused element.

**MAS Reference:**
MAS2.4.7 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={3b98b04f-4fab-4d3e-81af-a0906a34849c}

**Attachment For Reference:**
[A11y_VSCode_StatusBar_Keyboard_NotificationButtonsFocusVisibility.pptx](https://github.com/Microsoft/vscode/files/2127505/A11y_VSCode_StatusBar_Keyboard_NotificationButtonsFocusVisibility.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by bpasero** [comment]
@AccessibilityTestingTeam-TCS we have a high contrast theme for people that want a much stronger focus indication, isn't that sufficient?

![image](https://user-images.githubusercontent.com/900690/41890472-e7be8336-790f-11e8-8453-2666abc1cbd1.png)

**Comment by AccessibilityTestingTeam-TCS** [comment]
Agreed focus visibility is working for High contrast theme. But all low vision users may not be comfortable in using only one theme(here High Contrast). Other users using "Light Visual Studio" will again face issues in focus visibility. Switching themes may not be always a good option. Dotted-lines/rectangle to indicate focus can serve purpose for all of the themes listed.

**Comment by bpasero** [comment]
There is a border around the button when it gets focus but the border (blue) color collides with the blue background of the button. It may work better in other themes.

**Comment by isidorn** [comment]
@bpasero do we plan to do something here? If no we should close as invalid and explain why
@misolori fyi since it is a contrast issue

**Comment by bpasero** [comment]
I am happy for suggestions.

**Comment by misolori** [comment]
@bpasero we did start showing the outline with an offset in the extension viewlet:

<img width="420" alt="image" src="https://user-images.githubusercontent.com/35271042/45966367-c47def80-bfdf-11e8-9eda-f774c6e365ea.png">

We could try the same thing with our primary buttons:

<img width="477" alt="image" src="https://user-images.githubusercontent.com/35271042/45966422-f1ca9d80-bfdf-11e8-8135-6a9825588722.png">

**Comment by bpasero** [comment]
@misolori I actually like that if we can make it so that it really only shows up on focus but e.g. not when clicking (this might be a bit tricky to get right).

**Comment by isidorn** [comment]
Gives me a bit of a blurry effect. Definetly would not like to get it on click.
THough might be one of those things that I need to really try out to get a complete impression
